test/tests/functional/pbs_root_owned.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
```

Comment:
It would be better if file name changed to 'pbs_root_owned_script.py' or 'pbs_root_owned_job_script.py'